src/main/fc/rc_controls.c

```diff
@@ -118,11 +119,20 @@ throttleStatus_e calculateThrottleStatus(void)
     return THROTTLE_HIGH;
 }
 
+#define ARM_DELAY_TIME 500000 //arm,disarm delay usec
+#define STICK_DELAY_RATIO 10  //stick command delay usec = ARM_DELAY_TIME/STICK_DELAY_RATIO
 void processRcStickPositions(throttleStatus_e throttleStatus)
 {
-    static uint8_t rcDelayCommand;      // this indicates the number of time (multiple of RC measurement at 50Hz) the sticks must be maintained to run or switch off motors
-    static uint8_t rcSticks;            // this hold sticks position for command combos
-    static uint8_t rcDisarmTicks;       // this is an extra guard for disarming through switch to prevent that one frame can disarm it
+    // RC refresh rate converted to number of time the sticks must be maintained to arm,disarm
+    uint8_t rcDelayTicks = constrain(ARM_DELAY_TIME / getTaskDeltaTime(TASK_RX), 20, 50);
+    // indicates the number of time the sticks are maintained
+    static uint8_t rcDelayCommand;
```

Comment:
Let's extract this bunch of static variables to global scope to save some time on checking whether they've been initialized at runtime.